base/compiler/ssair/inlining.jl

```diff
@@ -1033,6 +1082,36 @@ function process_simple!(ir::IRCode, todo::Vector{Pair{Int, Any}}, idx::Int, sta
         return nothing
     end
 
+    # Inlining OpaqueClosure that don't have any specializations
+    if sig.ft ⊑ Core.OpaqueClosure
+        callee = stmt.args[1]
+        if isa(callee, SSAValue) && isexpr(ir.stmts[callee.id][:inst], :new_opaque_closure) &&
+                length(ir.stmts[callee.id][:inst].args) >= 5 &&
+                isa(ir.stmts[callee.id][:inst].args[4], OpaqueClosureIdx)
+            ir′ = ir.opaques[(ir.stmts[callee.id][:inst].args[4]::OpaqueClosureIdx).n]::IRCode
+            push!(todo, idx=>InliningTodo(nothing, ResolvedInliningSpec(ir′, linear_inline_eligible(ir′)), true))
+            return nothing
```

Comment:
DRY style: values should not be recomputed unnecessarily (and long lines often split by introducing named intermediates)
```suggestion
        if isa(callee, SSAValue)
             callee_inst = ir.stmts[callee.id][:inst]
             if isexpr(callee_inst, :new_opaque_closure) && length(callee_inst.args) >= 5
                idx = callee_inst.args[4]
                if isa(idx, OpaqueClosureIdx)
                    ir′ = ir.opaques[idx.n]::IRCode
                    todo = InliningTodo(nothing, ResolvedInliningSpec(ir′, linear_inline_eligible(ir′)), true))
                    push!(todo, idx => todo)
                    return nothing
                end
            end
```

Reply:
Why `>= 5`? That should have a comment. (We know that `length(callee_inst.args) >= 4` is guaranteed by the IR validator, so we don't need to assert that anywhere else)